Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -263,6 +263,8 @@ public abstract class ApplicationHost : IServerApplicationHost, IDisposable
         /// <inheritdoc/>
         public bool ListenWithHttps => Certificate is not null && ConfigurationManager.GetNetworkConfiguration().EnableHttps;
 
+        public bool EnableMetrics => ConfigurationManager.Configuration.EnableMetrics || this._startupConfig.GetValue<bool>(EnableMetricsKey);
```

Comment:
Also i dont like the location of this property. FeatureFlags should not be aggregated in the Host class. Why not keep it in the ConfigManager and extend the check there?